lib/doekbase/data_api/tests/examples/get_mRNAs_for_gene_to_GFF.py

```diff
@@ -1,18 +1,17 @@
 import doekbase.data_api
 from doekbase.data_api.taxonomy.taxon.api import TaxonAPI
-from doekbase.data_api.annotation.genome_annotation import GenomeAnnotationAPI
+#from doekbase.data_api.annotation.genome_annotation import GenomeAnnotationAPI
+from doekbase.data_api.annotation.genome_annotation.api import GenomeAnnotationAPI
 from doekbase.data_api.sequence.assembly.api import AssemblyAPI
 from doekbase.data_api.core import ObjectAPI
 import time
+import os
 
 services = {"workspace_service_url": "https://ci.kbase.us/services/ws/",
             "shock_service_url": "https://ci.kbase.us/services/shock-api/"}
 
 
-import os
-token = os.environ["KB_AUTH_TOKEN"]
-
-def get_gff(gene):
+def get_gff(gene, genome_annotation):
```

Comment:
It seems like you don't really need the function get_gff() in get_mRNAs_for_gene_to_GFF.py